Tasks/IISWebAppManagementOnMachineGroup/IISWebAppManagementOnMachineGroup.ps1

```diff
@@ -78,6 +79,12 @@ $appPoolPassword = Get-VstsInput -Name "AppPoolPassword"
 # Additional appcmd inputs 
 $appCmdCommands = Get-VstsInput -Name "AppCmdCommands"
 
+if($enableIIS -eq "true")
+{
+    import-module servermanager
+    add-windowsfeature web-server -includeallsubfeature
```

Comment:
Better to use explicit list of sub-features which you earlier prepared.

Reply:
As discussed we'll use --includeallsubfeatres